backend/index.js

```diff
@@ -49,6 +49,33 @@ app.get('/', (req, res) => {
   })
 
 
+  app.post('/exercises/:id/sets', async (req, res) => {
+    const { id } = req.params;
+    const { weight, reps } = req.body
+    const newSet = await prisma.set.create({
+      data: {
+        weight,
+        reps,
+        exerciseId: parseInt(id)
```

Comment:
nit: maybe use Number() instead of parseInt() as Number(id) will throw a SyntaxError, which is more informative than getting NaN from parseInt().

Reply:
Sounds good